lib/App/cpanminus/script.pm

```diff
@@ -91,6 +91,7 @@ sub new {
         installed_dists => 0,
         install_types => ['requires'],
         with_develop => 0,
+        with_configure => undef,
```

Comment:
That was my original approach. However the current behaviour, is if nothing is set then configure dependencies should be installed when dependencies are read from META files, but not when read from cpanfile. This required 3 possible states:-

True : install configure dependencies
False: do not install configure dependencies
undef: install configure dependencies when dependencies are read from META files, but not when read from cpanfile

I'm happy to amend things however.